packages/chakra-ui/src/Theme/Theme.tsx

```diff
@@ -0,0 +1,17 @@
+import ArrayFieldTemplate from '../ArrayFieldTemplate'
+import ErrorList from '../ErrorList'
+import Fields from '../Fields'
+import FieldTemplate from '../FieldTemplate'
+import ObjectFieldTemplate from '../ObjectFieldTemplate'
+import Widgets from '../Widgets'
+
+const Theme = {
+  ArrayFieldTemplate,
+  fields: Fields,
+  FieldTemplate,
+  ObjectFieldTemplate,
+  widgets: Widgets,
+  ErrorList
```

Comment:
Can you create a `SubmitButton` entry and use the chakra-ui-themed button for it? (example: https://github.com/rjsf-team/react-jsonschema-form/blob/fd107fbd3170fea22761652bef94732daf57f9a9/packages/fluent-ui/src/Theme/Theme.ts)

Right now, the default button doesn't look right:

![image](https://user-images.githubusercontent.com/1689183/111926776-911f5d00-8a84-11eb-8bb9-40ac591edb93.png)